src/frontend/game/components/IncreaseMyScore.tsx

```diff
@@ -14,5 +14,9 @@ export const IncreaseMyScore: FunctionalComponent = () => {
 		});
 	};
 
-	return <button onClick={onClick}>increase my score by 100</button>;
```

Comment:
if you're using this pr to learn how to create a new game event this file can be ignored, the changes are unrelated to implementing the ChangePlayerNameEvent.